### PR TITLE
Logging error onto error_log

### PR DESCRIPTION
**Pull request by jiskanulo**
- logging console->log() to error_log
- ignore browser client logging on production

### CODE REVIEW

File: Samurai/Samurai/Component/Console/Client/BrowserClient.php [hunk 1]
```diff
@@ -61,7 +61,13 @@ class BrowserClient extends Client implements Optimizer
      */
     public function send($level, $message)
     {
-        $this->messages[] = ['level' => $level, 'message' => $message];
+        if (! $this->application->isProduction()) {
+            $this->messages[] = ['level' => $level, 'message' => $message];
+        }
+
+        if ($level >= self::LOG_LEVEL_WARN) {
+            error_log($this->wrapping($message));
```
Comment (8823-scholar) [inline Samurai/Samurai/Component/Console/Client/BrowserClient.php]: おっと、これは、MultipleClientを使用する事前提で、ErrorLogClient.phpなりを用意したほうがいいね＞＜

oh, create the ErrorLogClient.php on the assumption that use MultipleClient.php is better than use BrowserClient.php

**Comment by jiskanulo** [comment]
updated

After this PR, I plan to set ErrorLogClient in Initializer.

**Comment by 8823-scholar** [comment]
@jiska thx!

**Comment by jiskanulo** [comment]
:heart:

**Pull request merged**